sass/_theme_font_local.sass

```diff
@@ -12,27 +12,32 @@
 
 @font-face
   font-family: 'Lato'
-  font-style: normal
+  src: url('../fonts/Lato/lato-regular.eot')
+  src: url('../fonts/Lato/lato-regular.ttf') format('truetype'), url('../fonts/Lato/lato-regular.svg#latoregular') format('svg')
   font-weight: 400
-  src: local('Lato Regular'), local('Lato-Regular'), url(../fonts/Lato-Regular.ttf) format('truetype')
+  font-style: normal
 
 @font-face
   font-family: 'Lato'
-  font-style: normal
+  src: url('../fonts/Lato/lato-bold.eot')
+  src: url('../fonts/Lato/lato-bold.eot?#iefix') format('embedded-opentype'), url('../fonts/Lato/lato-bold.woff2') format('woff2'), url('../fonts/Lato/lato-bold.woff') format('woff'), url('../fonts/Lato/lato-bold.ttf') format('truetype'), url('../fonts/Lato/lato-bold.svg#latobold') format('svg')
   font-weight: 700
-  src: local('Lato Bold'), local('Lato-Bold'), url(../fonts/Lato-Bold.ttf) format('truetype')
+  font-style: normal
+
 
 @font-face
   font-family: 'Lato'
+  src: url('../fonts/Lato/lato-bolditalic.eot')
+  src: url('../fonts/Lato/lato-bolditalic.eot?#iefix') format('embedded-opentype'), url('../fonts/Lato/lato-bolditalic.woff2') format('woff2'), url('../fonts/Lato/lato-bolditalic.woff') format('woff'), url('../fonts/Lato/lato-bolditalic.ttf') format('truetype'), url('../fonts/Lato/lato-bolditalic.svg#latobold_italic') format('svg')
```

Comment:
I don't believe we use bold and italic in the theme anywhere. Is that wrong? This will result in additional disk usage for no benefit although that isn't a huge deal. Regardless, I'd say we shouldn't include it unless there's a reason.

Reply:
It's not possible to express bolditalic in rst syntax directly, but there are situations where it can happen, like bold text in a caption (which is italic by default) or italic text in a table header (which is already bold by default):

```rst
.. list-table::
    :header-rows: 1

    * - header
      - **bold header**
      - *italic header*
    * - cell
      - **bold cell**
      - *italic cell*
```

is rendered as:
<img width="338" alt="table with italic header" src="https://user-images.githubusercontent.com/3284111/40107398-6683c646-58f8-11e8-8b33-e4434a35b10f.png">